notebook/services/kernels/kernelmanager.py

```diff
@@ -71,6 +71,16 @@ def _update_root_dir(self, proposal):
         help="""The interval (in seconds) on which to check for idle kernels exceeding the cull timeout value."""
     )
 
+    cull_connected = Bool(True, config=True,
+        help="""Whether to consider culling kernels which have one or more connections.
+        Only effective if cull_interval is not 0."""
```

Comment:
Do you mean `cull_idle_timeout` in both of these messages? Looking at the code, setting `cull_interval` to 0 produces a warning and resets it to 300.